CRM/Member/BAO/Membership.php

```diff
@@ -1852,12 +1852,6 @@ public static function processMembership($contactID, $membershipTypeID, $is_test
         if (!empty($membershipSource)) {
           $currentMembership['source'] = $membershipSource;
         }
-        else {
```

Comment:
I can demonstrate on the other function with a test that we don't need to pass this in if unchanged